packages/plugin-libet-intentional-binding/README.md

```diff
@@ -0,0 +1,39 @@
+# libet-intentional-binding plugin
+
+## Overview
+
+This plugin measures intentional binding using a Libet clock, and allows the participant to estimate the timing of events by adjusting the clock hand themselves.
+
+## Loading
+
+### In browser
+
+```js
+<script src="https://unpkg.com/@jspsych-contrib/plugin-libet-intentional-binding@1.0.0">
+```
+
+### Via NPM
+
+```
+npm install @jspsych-contrib/plugin-libet-intentional-binding
+```
+
+```js
+import jsPsychLibetIntentionalBinding from '@jspsych-contrib/plugin-libet-intentional-binding';
+```
+
```

Comment:
Oh, and the NPM part of this should be removed as the plugin isn't implemented as a module.

```suggestion
```js
<script src="https://unpkg.com/@jspsych-contrib/plugin-libet-intentional-binding@1.0.0">
```

```